test/util.ts

```diff
@@ -109,6 +110,9 @@ export function runBaselineTest(options: BaselineOptions) {
     if (options.iamService) {
       commandLine += ` --iam-service="${iamService}"`;
     }
+    if (options.metadata) {
```

Comment:
could be interesting to try [yargs-unparser](https://www.npmjs.com/package/yargs-unparser) here ... I don't use the library anywhere, so find I'm not great at keeping up-to-date, having a consumer would be a forcing function.

Reply:
Hm. This might be a bigger refactor (I'm not immediately sure the syntax of all parameters is the same when passed down) but I'll look into this!

Also, just to give a context, this file (`gapic-generator-typescript.ts`) is only useful for local development and running; it's not used in bazel pipeline (bazel fast tracks to the second stage, `protoc-plugin.ts`). It does not mean the argument parsing here cannot be improved :)